packages/go/analysis/ad/esc3.go

```diff
@@ -688,8 +688,8 @@ func ADCSESC3Path6_7Pattern(ecaId graph.ID, domainId graph.ID) traversal.Pattern
 			query.KindIn(query.Relationship(), ad.IssuedSignedBy, ad.EnterpriseCAFor),
 		)).
 		OutboundWithDepth(0, 0, query.And(
-			query.Kind(query.Start(), ad.EnterpriseCA),
 			query.KindIn(query.Relationship(), ad.IssuedSignedBy, ad.EnterpriseCAFor),
+			query.KindIn(query.End(), ad.EnterpriseCA, ad.AIACA, ad.RootCA),
```

Comment:
This pattern looks slightly different from the other places this has been updated. Are these blocks equivalent? 

```go
		OutboundWithDepth(0, 0, query.And(
			query.KindIn(query.Relationship(), ad.IssuedSignedBy, ad.EnterpriseCAFor),
			query.KindIn(query.End(), ad.EnterpriseCA, ad.AIACA, ad.RootCA),
		)).
		Outbound(query.And(
			query.KindIn(query.Relationship(), ad.RootCAFor),
			query.Equals(query.EndID(), domainId),
		))
```

```go
              OutboundWithDepth(0, 0, query.And(
			query.KindIn(query.Relationship(), ad.IssuedSignedBy, ad.EnterpriseCAFor),
			query.KindIn(query.End(), ad.EnterpriseCA, ad.AIACA),
			)).	
		Outbound(
			query.And(
				query.KindIn(query.Relationship(), ad.IssuedSignedBy, ad.EnterpriseCAFor),
				query.Kind(query.End(), ad.RootCA),
			)).
		Outbound(
			query.And(
				query.KindIn(query.Relationship(), ad.RootCAFor),
				query.Equals(query.EndID(), domainID),
			))
```

If they are equivalent, I would recommend writing them the same way but if they are not equivalent maybe add a comment as to how and why it is done differently here.

Reply:
Good point 👍